### PR TITLE
datasource/cb ut

### PR DESCRIPTION
**Pull request by sanxun0325**
### Describe what this PR does / why we need it
Supplementary unit testing

### Describe how to verify it
go test ./...

### Special notes for reviews

### CODE REVIEW

**Comment by sanxun0325** [comment]
The content is too much, so it can be reviewed first, and cannot be merged for the time being.  draft pr Settings were not found

**Comment by louyuting** [comment]
> The content is too much, so it can be reviewed first, and cannot be merged for the time being. draft pr settings were not found

Okay, the review might be later b/c holiday.

**Comment by louyuting** [comment]
@sanxun0325  Please let me know if the PR is ready.

**Comment by sanxun0325** [comment]
> @sanxun0325 Please let me know if the PR is ready.

ready

File: core/circuitbreaker/circuit_breaker_test.go [hunk 1]
```diff
@@ -1,9 +1,16 @@
 package circuitbreaker
 
 import (
+	"errors"
+	"sync/atomic"
 	"testing"
 
+	"github.com/alibaba/sentinel-golang/logging"
+
```
Comment (louyuting) [inline core/circuitbreaker/circuit_breaker_test.go:9]: Maybe you should refine imports(go imports)

**Review by louyuting — APPROVED** [review]
Nice work!

**Comment by sczyh30** [comment]
Cool 🥇

**Pull request merged**